AElf.Kernel/Services/IBlockchainService.cs

```diff
@@ -133,15 +150,32 @@ private async Task ExecuteBlock(int chainId, ChainBlockLink blockLink)
             await _chainManager.SetChainBlockLinkAsExecuted(chainId, blockLink);
         }
 
-
         public async Task<Block> GetBlockByHashAsync(int chainId, Hash blockId)
         {
             return await _blockManager.GetBlockAsync(blockId);
         }
 
+        public async Task<Block> GetBlockByHeightAsync(int chainId, ulong height)
+        {
+            var index = await _chainManager.GetChainBlockIndexAsync(chainId, height);
```

Comment:
it only provide LIB height, so maybe you should see the implements of release 0.9.0 
and also just implement IFullBlockchainService